cogs5e/gametrack.py

```diff
@@ -466,11 +468,24 @@ async def customcounter(self, ctx, name=None, *, modifier=None):
         await character.commit(ctx)
 
         if new_value - counter.value:
-            out = f"{str(counter)}\n({abs(new_value - counter.value)} overflow)"
+            overflow = new_value - counter.value
+            changed = counter.value - old_value
+            out = f"{str(counter)}\n({abs(overflow)} overflow)"
+            # Calculations for Change (with overflow)
+            if not operator or operator == 'mod':
```

Comment:
`operator` is guaranteed to exist here, so I don't think you need to do this check (tbf, my code is a little spaghetti. 😛)

Since one of the if bodies is guaranteed to be true, and they seem to have the same body, you might be able to take the delta code out and place it outside of the `if new_value - counter.value:`/`else` block?

Reply:
`operator` is None if the user doesn't specify `mod` or `set` (Ex `!cc testing -4`) So this check is for if it's not nothing or mod